splice_machine/src/main/java/com/splicemachine/derby/impl/store/access/hbase/HBaseConglomerate.java

```diff
@@ -364,39 +364,6 @@ protected void writeExternalOld(ObjectOutput out) throws IOException {
                 .build();
         return dvd;
     }
-    /**
-     * Restore the in-memory representation from the stream.
-     * <p/>
-     *
-     * @throws ClassNotFoundException Thrown if the stored representation
-     *                                is serialized and a class named in
-     *                                the stream could not be found.
-     * @see java.io.Externalizable#readExternal
-     **/
-    @Override
-    public void readExternal(ObjectInput in) throws IOException {
```

Comment:
DB-11857 modifies HBaseConglomerate.readExternal, and then DB-11890 delete exactly that function, and i'm not sure how to resolve that. Are you sure HBaseConglomerate.readExternal should be deleted?